src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -182,7 +181,7 @@ public static void info(String description, Throwable e) {
   public static void image(String command, File image, boolean success) {
     byte[] bytes = new byte[0];
     try {
-      bytes = new Base64().encode(FileUtils.readFileToByteArray(image));
+      bytes = Base64.getEncoder().encode(FileUtils.readFileToByteArray(image));
     } catch (IOException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)